### PR TITLE
chore: build pause container for `run local`

### PR DESCRIPTION
**Pull request by dannyrandall**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/docker/orchestrator/orchestrator.go [hunk 1]
```diff
@@ -43,15 +48,22 @@ type DockerEngine interface {
...
-const (
-	pauseContainerURI = "public.ecr.aws/amazonlinux/amazonlinux:2023"
+const pauseContainerURI = "aws-copilot-pause"
+
+var (
+	// '+' is not a valid character in an image tag
+	pauseContainerTag = strings.Replace(version.Version, "+", "PLUS", -1)
+
+	//go:embed Pause-Dockerfile
+	pauseDockerfile string
```
Comment (KollaAdithya) [inline internal/pkg/docker/orchestrator/orchestrator.go]: I assume we can use `os.ReadFile(filePath)` to read the content of pause container Dockerfile. Is there any reason for using the embed directive.

File: internal/pkg/docker/orchestrator/orchestrator.go [hunk 2]
```diff
@@ -43,15 +48,22 @@ type DockerEngine interface {
...
 	pauseCtrTaskID            = 0
 )
 
-const (
-	pauseContainerURI = "public.ecr.aws/amazonlinux/amazonlinux:2023"
+const pauseContainerURI = "aws-copilot-pause"
+
+var (
+	// '+' is not a valid character in an image tag
+	pauseContainerTag = strings.Replace(version.Version, "+", "PLUS", -1)
```
Comment (iamhopaul123) [inline internal/pkg/docker/orchestrator/orchestrator.go]: wut do we need this for ease of development? Why do we even need to build the img with a tag 🤔

**Review by iamhopaul123 — APPROVED** [review]

File: internal/pkg/docker/orchestrator/orchestrator.go (hunk 1 above)
Comment (dannyrandall) [reply internal/pkg/docker/orchestrator/orchestrator.go]: yeah! `os.ReadFile` reads a file _on the users machine._ `//go:embed` embeds the file _in to the binary itself._ We don't expect users to have the `Pause-Dockerfile` available on their machine, so we are including it in the binary.

File: internal/pkg/docker/orchestrator/orchestrator.go (hunk 2 above)
Comment (dannyrandall) [reply internal/pkg/docker/orchestrator/orchestrator.go]: yeah it is only development🥺 i think you're right - i'll just use the `latest` tag always instead 😊

**Review by KollaAdithya — APPROVED** [review]
lgtm! :shipit:

**Pull request merged**